docs/source/tutorial/contingent-claims-on-vs-off-ledger.rst

```diff
@@ -0,0 +1,47 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Tutorials : Contingent Claims tree on vs off ledger
+###################################################
+
+Different ways to create and store the Contingent Claims tree
+*************************************************************
+
+We have now seen two different ways of modeling a fixed coupon bond using ``Contingent Claims``:
+
+On ledger
+=========
+
+When we use the :doc:`Derivative extension <derivative-extension>` we create the
+claims tree at instrument inception and store this representation directly
+on the ledger. Since the tree is stored statically it can only change if the instrument is
+updated on ledger. For example, after a coupon payment a new version of the instrument (excluding
+the coupon just paid) supersedes the previous version.
+However, in the event of a change in a holiday calendar (which could be used to define the
+coupon payment dates), the tree will not automatically change.
+
+Off ledger
```

Comment:
I wouldn't call this "off-ledger" as we're not really doing anything off-ledger here. Rather, we create the claims tree on-the-fly. So maybe the two options are
- Explicitly storing the contingent-claims tree on the ledger
- Calculating the contingent-claims tree on-the-fly

Reply:
Good point, I have changed this now.